src/components/FilterBuilds.vue

```diff
@@ -124,11 +194,62 @@ export default {
       this.$data.rdate = date.getTime() / 1000
       console.log('final ' + this.$data.rdate)
     },
-    dt: function(timestamp) {
-      return new Date(timestamp * 1000)
+    formatDate: function(timestamp) {
+      var d = new Date(timestamp * 1000)
```

Comment:
moment is supposed to help for this kind of code.
You should be able to use it by requiring MOMENT in the angularjs dependency injection of your directive.
You can also use DurationFormat and DateFormat for a standard way of formating dates in buildbot

https://github.com/buildbot/buildbot/blob/d913f1ec10dc7b751d414c090b83644c192d85a4/www/base/src/app/common/filters/moment/moment.filter.js#L13

They are also available as dependency injection.

Reply:
Okay, I will change that in the next pr